timemachine/fe/mcgregor.py

```diff
@@ -128,6 +128,30 @@ def __init__(self, n_vertices, edges):
             for edge_idx in edges:
                 self.ve_matrix[vertex_idx][edge_idx] = True
 
+    def _verify_partially_connected(self, source, node_states, n_mapped_nodes):
+        """A fast BFS node generator, adapted from _plain_bfs in networkx."""
```

Comment:
nit: Might be good to add a permalink to the networkx code for future reference

Reply:
nit: this docstring seems like an incomplete description. I'm not sure I understand what "partially" means here?

Reply:
willfix